src/date-picker/index.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component, PropTypes } from 'react';
+import DateTime from 'react-datetime';
+
+// This component really just extends react-datetime
+// found at https://github.com/arqex/react-datetime.
+
+
+// import './date-picker.scss';
```

Comment:
you can wrap it in a div, and modulize the wrapper class -- see patternity-evo/src/react-select